nodejs/eks/cluster.ts

```diff
@@ -341,6 +341,7 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
 
     // Create the EKS cluster
     const eksCluster = new aws.eks.Cluster(`${name}-eksCluster`, {
+        name: args.name ? name : undefined,
```

Comment:
Shouldn't this just be `args.name` instead of `args.name ? name : undefined`?  Pretty sure as is this will not do what it says it does.

Reply:
Good catch. I've over thought this PR further than required 😛